Examples/test-suite/r/Makefile.in

```diff
@@ -25,7 +25,8 @@ CPP_TEST_CASES += \
 	r_overload_array \
 	r_sexp \
         r_overload_comma \
-	r_use_isnull
+	r_use_isnull \
+	li_boost_shared_ptr
```

Comment:
`li_boost_shared_ptr` should already be in `CPP_TEST_CASES` (from `Examples/test-suite/common.mk`) - only R-specific testcases should need adding here.

Reply:
ok, should be fixed in the commit "fixes from code review"